src/engine/pieces/rook.ts

```diff
@@ -1,13 +1,13 @@
-import Piece from './piece';
+import Piece, {PieceType} from './piece';
```

Comment:
nitpick but is's convention to put spacing around things in curly braces 
e.g. `{ PieceType }`